manifest.json

```diff
@@ -9,7 +9,7 @@
   "applications": {
     "gecko": {
       "id": "{e4a8a97b-f2ed-450b-b12d-ee082ba24781}",
-      "strict_min_version": "52.0"
+      "strict_min_version": "57.0"
```

Comment:
Why?

Reply:
[filterResponseData](https://github.com/Sxderp/greasemonkey/blob/3c00f700159212296dcbf968bfa043ca0af3af6c/src/bg/user-script-detect.js#L49) is FF 57+
https://developer.mozilla.org/en-US/Add-ons/WebExtensions/API/webRequest/filterResponseData